### ISSUE TITLE
Pdftotext crashes ranger

### ISSUE THREAD

**Issue by Gomme-** [ranger/ranger#1251]
#### Runtime Environment
Operating system and version: Archlinux, kernel 4.17.13(.arch1-1)
Terminal emulator and version: st 0.8.1
ranger version: ranger-master 1.9.1
Python version: 3.7.0 (default, Jul 15 2018, 10:44:58) [GCC 8.1.1 20180531]
Locale: fr_FR.UTF-8

#### Current Behavior
Recently pdf preview (pdftotext) crashes ranger. No problem when I remove the pdftotext line in scope.sh.
I can't remember if it started after some ranger or python archlinux update...

#### Traceback
ranger version: ranger-master 1.9.1
Python version: 3.7.0 (default, Jul 15 2018, 10:44:58) [GCC 8.1.1 20180531]
Locale: fr_FR.UTF-8
Current file: '/home/user/Documents/Bibliothèque/some_pdf.pdf'

Traceback (most recent call last):
```
File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/pager.py", line 236, in _generate_lines
    raise StopIteration
StopIteration

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/ranger/core/main.py", line 196, in main
    fm.loop()
  File "/usr/lib/python3.7/site-packages/ranger/core/fm.py", line 390, in loop
    ui.redraw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/ui.py", line 338, in redraw
    self.draw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/ui.py", line 365, in draw
    DisplayableContainer.draw(self)
  File "/usr/lib/python3.7/site-packages/ranger/gui/displayable.py", line 256, in draw
    displayable.draw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/view_miller.py", line 100, in draw
    DisplayableContainer.draw(self)
  File "/usr/lib/python3.7/site-packages/ranger/gui/displayable.py", line 256, in draw
    displayable.draw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/browsercolumn.py", line 176, in draw
    self._draw_file()
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/browsercolumn.py", line 203, in _draw_file
    Pager.draw(self)
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/pager.py", line 100, in draw
    for line, i in zip(line_gen, range(self.hei)):
RuntimeError: generator raised StopIteration

ranger crashed. Please report this traceback at:
https://github.com/ranger/ranger/issues
```

**Comment by mxovd** [comment]
This is already fixed in  #1244 , you can clone the latest git

**Comment by toonn** [comment]
Yep, breaking change in python 3.7, either wait for the next release, run master (ranger-git in the AUR) or downgrade python.
